aioconsole/execute.py

```diff
@@ -50,6 +52,12 @@ def make_coroutine_from_tree(tree, filename="<aexec>", symbol="single", local={}
     return dct[CORO_NAME](**local)
 
 
+def get_non_indented_lines(source):
+    for token in generate_tokens(StringIO(source).readline):
+        if token.type == STRING:
+            yield from range(token.start[0], token.end[0])
```

Comment:
I think it's more clear if we `yield from range(token.start[0] + 1, token.end[0] + 1)` (considering all lines but the first one should be indented), and then `enumerate(source.split("\n"), start=1)` with a comment noting that the line numbers are one-indexed.

Reply:
I don't see why do we need it. Would it be ok just to add comment to existing code?

Reply:
Sure, as long as it's clear for my future self running into this piece of code in 2 years wondering "why do include the first line of the string token as a non-indented line?" :)